Tests/DependencyInjection/FOSRestExtensionTest.php

```diff
@@ -684,4 +684,48 @@ public function testExceptionThrownIfCallbackFilterIsUsed()
     {
         $this->extension->load(array('fos_rest' => array('view' => array('jsonp_handler' => array('callback_filter' => 'foo')))), $this->container);
     }
+
+    public function testParamConverterClassParameterWithFrameworkExtraBundle20()
+    {
+        // skip the test if the installed version of SensioFrameworkExtraBundle
+        // is not compatible with the RequestBodyParamConverter20 class
+        $parameter = new \ReflectionParameter(
+            array(
+                'Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface',
+                'supports',
+            ),
+            'configuration'
+        );
+        if ('Sensio\Bundle\FrameworkExtraBundle\Configuration\ConfigurationInterface' != $parameter->getClass()->getName()) {
+            $this->markTestSkipped(
+                'skipping RequestBodyParamConverter20Test due to an incompatible version of the SensioFrameworkExtraBundle'
+            );
+        }
+
+        $this->extension->load(array('fos_rest' => array('body_converter' => array('enabled' => true))), $this->container);
+
+        $this->assertSame('FOS\RestBundle\Request\RequestBodyParamConverter20', $this->container->getParameter('fos_rest.converter.request_body.class'));
+    }
+
+    public function testParamConverterClassParameterWithFrameworkExtraBundle3()
```

Comment:
Rather than having 2 tests with one of them being skipped each time and only changing the expectation, I suggest using a single test changing the expectation based on the bundle version